test/ServerRequestFactoryTest.php

```diff
@@ -20,6 +20,18 @@
 
 class ServerRequestFactoryTest extends TestCase
 {
+    protected $globalServer;
+
+    protected function setUp()
+    {
+        $this->globalServer = $_SERVER;
+    }
+
+    protected function tearDown()
+    {
+        $_SERVER = $this->globalServer;
+    }
+
```

Comment:
The test `testFromGlobalsUsesCookieSuperGlobalWhenCookieHeaderIsNotSet()` was causing inconsistent results due to superglobal state otherwise. This is better alleviated by the combination of `@runInSeparateProcess` and `@preserveGlobalState disabled` annotations.